README.md

```diff
@@ -18,7 +18,8 @@
 Before you start make sure you have downloaded and installed [Node.js LTS](https://nodejs.org/en/download/), [Yarn](https://yarnpkg.com/lang/en/docs/install/) and git.
 
 -   `git clone git@github.com:trezor/trezor-suite.git`
--   `yarn && yarn build:libs`
+-   `yarn`
+-   `yarn build:libs && yarn workspace @trezor/suite-data msg-system-sign-config`
```

Comment:
@trezor/trezor-suite This just got merged which means you'll likely get a failure of `copy-static-files` task once you pull develop and run suite locally. To remedy it just run `yarn workspace @trezor/suite-data msg-system-sign-config` which will generate the needed files.